src/lib.rs

```diff
@@ -27,6 +27,7 @@ mod stream;
 
 pub use crate::connection::{Connection, Mode, StreamHandle};
 pub use crate::error::{DecodeError, ConnectionError};
+pub use crate::stream::State;
```

Comment:
This is simply because I discovered in the API docs that `state(&self) -> Option<State>` is a publicly exported function of `StreamHandle`, but `State` was not visible.